src/SDKs/IotCentral/IotCentral.Tests/IotCentral.Tests.csproj

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <PackageId>IotCentral.Tests</PackageId>
-    <Version>1.0.0</Version>
+    <Version>1.1.0</Version>
```

Comment:
nit: this doesn't need to be updated, test projects aren't packed into a nupkg where the version would be shown. leaving this as 1.0.0 would be fine

Reply:
Reverted